setup/scripts/build_docker.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
```

Comment:
Perhaps it's not a big deal, but the file naming here is not consistent.  Do you want to make it so as `docker_builder.sh`?

Reply:
Reverted to your original suggestion: `build_docker.sh` matching the naming convetion of the other scripts.